index.html

```diff
@@ -1,252 +1,252 @@
-                    <p class="p2">|</p>
-                </div>
-                <div class="contact-info3">
-                    <p class="p2">Enter your message here...</p>
-                </div>
-                <div  class="get-startedcollaboration">
-                <p class="get-started-start-collab">start collaboration <p>
-                </div>
-                <div class="contact-img">
-                    <button class="resume-btn"><a href="#"> Get my Resume  <img class="share-icon" src="images/icons8-share-24.png" alt="union"></a></button>
-                </div>
+                <form action="https://formspree.io/f/xgebyodb" method="post">
+                    <input type="text" class="name" name="name" placeholder="your name" required="true" maxlength="30" > <br>
+                    <input type="email" class="email" name="Email" required="true" placeholder="your email">
+                    <textarea name="text-area" id="your-message" rows="5" class="message" required="true" placeholder="Enter your message..."></textarea>
+                    <p class="character-limit">0/500</p>
+                    <div class="btn-container">
+                    <button class="start-collab" type="submit">start collaboration</button>
+                    </div>                 
                 
-                    
-                <div class="icons">
+            </form>            
+            <div class="contact-img">
+                <button class="resume-btn"><a href="#"> Get my Resume  <img class="share-icon" src="images/icons8-share-24.png" alt="union"></a></button>
+            </div>
+            <div class="icons">
                     <ul>
-                        <li><a href="#"><img src="images/ic_stackoverflow.png" alt=""></a><li> 
-                        <li><a href="#"><img src="images/githubicon.png" alt=""></a><li>
-                        <li><a href="#"><img src="images/twittericon.png" alt=""></a><li>
-                        <li><a href="#"><img src="images/linkedin.png" alt=""></a><li>
-                        <li><a href="#"><img src="images/medium (2).png" alt=""></a><li>
-                        <li><a href="#"><img src="images/angellist (2).png" alt=""></a><li>
-                    </ul>
-                </div>                    
-            </div>
-        </section>
-        
-        
\ No newline at end of file
+                    <li><a href="#"><img src="images/ic_stackoverflow.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/githubicon.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/twittericon.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/linkedin.png" alt=""></a><li>
+                    <li><a href="#"><img src="images/medium (2).png" alt=""></a><li>
+                    <li><a href="#"><img src="images/angellist (2).png" alt=""></a>
+                    <li>
+                </ul>
```

Comment:
- [ ] So far almost everything is working well 👏 👍 . However, we need to ensure that the portfolio is professional. If you could have met some of the articles, the traditional horizontal scrollbar burdens the user by requiring constant attention and greater physical effort to maintain the dragging 😞 . Therefore kindly ensure that there is no horizontal scroll effect ☺️ .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/222106583-72e5d9be-113b-44b0-9338-3657aec0d110.png)